mvpoly/src/monomials.rs

```diff
@@ -447,6 +448,106 @@ impl<const N: usize, const D: usize, F: PrimeField> MVPoly<F, N, D> for Sparse<F
     fn from_expr<Column: Into<usize>>(expr: Expr<ConstantExpr<F>, Column>) -> Self {
         prime::Dense::from_expr(expr).into()
     }
+
```

Comment:
```suggestion
/// see https://hackmd.io/@dannywillems/Syo5MBq90
```

Reply:
The documentation is already added from the trait.